profiles/common/modules/custom/nexteuropa_piwik/nexteuropa_piwik.install

```diff
@@ -91,4 +91,111 @@ function nexteuropa_piwik_enable() {
   );
   // Grant the permissions to PIWIK administrator role.
   user_role_change_permissions($piwik_admin_role->rid, $piwik_admin_permissions);
+
+  $administrator = user_role_load_by_name('administrator');
```

Comment:
This code is the same that used here : https://github.com/ec-europa/platform-dev/pull/1259/files#diff-7ce62a7971f302457a6782b94566fac3R191
Maybe we can use a custom function to do this action.